chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/PrecacheableDimensionalDataManager.java

```diff
@@ -193,12 +193,13 @@ public PrecacheableDimensionalDataManager(
     }
 
     @Transactional
-    @Cacheable(CacheNames.DEVELOPER_NAMES)
+    @Deprecated
```

Comment:
Seems like this file is misnamed. It's not "precached" anymore, right? Should it be "CacheableDimensionalDataManager"?

Reply:
Refactored.